api/controllers/site.js

```diff
@@ -160,51 +160,52 @@ module.exports = {
     const siteId = Number(req.params.id);
 
     Promise.resolve(siteId).then((id) => {
-      if (isNaN(id)) {
+      if (_.isNaN(id)) {
```

Comment:
s this necessary? this style isNaN is not used throughout the codebase

Reply:
The isNaN was throwing linting error's so I used the underscore version